environment/src/main/java/com/sequenceiq/environment/credential/service/ServiceProviderCredentialAdapter.java

```diff
@@ -43,6 +43,9 @@
 @Component
 public class ServiceProviderCredentialAdapter {
 
+    public static final String FAILED_CREDETIAL_VERIFICATION_MESSAGE = "Failed to verify the credential "
+            + "(try later if policies and roles are newly created or modified): ";
```

Comment:
Maybe we could be a little bit more precise about how much should one wait until retry their request. "later" can be a couple of mins or a day.. I mean only if we can be more precise with a rough estimation like "a couple of minutes later" or something like this. What do you think?

Reply:
honestly don't know. I've got many similar messages without further indicator about the time frame, so I thought it will be alright here as well